examples/sample_files/sample.md

```diff
@@ -0,0 +1,65 @@
+---
+type: intro
+date: 1.1.2023
+---
+```bash
+pip install farm-haystack
```

Comment:
I know this is just an example document but I can also see how it can create problems from us and confusion on discord 🤣 

Let's switch it to haystack-ai?